plugins/mapper-murmur3/src/yamlRestTest/resources/rest-api-spec/test/mapper_murmur3/10_basic.yml

```diff
@@ -104,3 +104,26 @@ setup:
           fields: [ foo.hash ]
 
   - match: { hits.hits.0.fields: {foo.hash: ["foo"]} }
+
+---
+"Murmur3 docvalue_fields api":
```

Comment:
Maybe we also need a test with synthetic source enabled?  The `test` index here doesn't synthetic source enabled. Just to double check murmur field works as expected.

Reply:
My thinking was that it does not matter but the test would serve as a documentation for that, i'll add it.